src/agent/ISmartReader.h

```diff
@@ -1,10 +1,12 @@
-
 #pragma once
+
 #include <string>
+#include "common/GeneralHealth.h"
 
 class ISmartReader
 {
 public:
 	virtual ~ISmartReader() = default;
 	virtual std::string ReadSMARTData() = 0;
+	virtual GeneralHealth ReadDisksStatus() = 0;
```

Comment:
I believe there should be another tool for calculating health. Now SmartReader will have two responsibilities 
which makes it harder to reuse.

Reply:
This will be changed in future task.